transit/transit_types.cpp

```diff
@@ -204,7 +204,7 @@ bool Gate::IsEqualForTesting(Gate const & gate) const
 
 bool Gate::IsValid() const
 {
-  return m_featureIdentifiers.GetOsmId() != kInvalidOsmId && m_weight != kInvalidWeight &&
+  return m_featureIdentifiers.GetSerialId() != kInvalidOsmId && m_weight != kInvalidWeight &&
```

Comment:
`kInvalidOsmId` is correct?

Reply:
I think that yes. OsmId is an alias to uint64_t in this file.